### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by leobalter**
I've got updates addressing (_trying to address_) several open issues: #3, #4, #9, #11, #12, #14, #15.

### CODE REVIEW

**Comment by leobalter** [comment]
cc @gnarf, I'll wait for your review and push it soon. If it's fairly ok, I believe we should try to address more feedback in the current document as a whole, before we have 2 drafts (master and here) and more feedback coming in during the next meeting.

File: Conduct.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+This isn't an exhaustive list of things that you can't do. Rather, take it in the spirit in which it's intended - a guide to make it easier to enrich all of us and the technical communities in which we participate and empower others to speak.
 
-This Code of Conduct applies to all spaces managed by TC39. This includes IRC, mailing lists, issue trackers, TC39 events, and any other forums created by TC39 delegates which the community uses for communication. In addition, violations of this code outside these spaces may affect a person's ability to participate within them.
+This Code of Conduct applies to all spaces managed by TC39. This includes IRC channels moderated by the TC39, mailing lists as the es-discuss, issue trackers on projects hosted by TC39, TC39 events as the meetings and others hosted by or representing TC39. In addition, violations of this code outside these spaces by a person others known to be a TC39 member is an act of representing TC39 may affect a person's ability to participate within them.
```
Comment (littledan) [inline Conduct.md]: Slight wording suggestion: did you mean to write "which" before "is"?

Comment (gnarf) [inline Conduct.md]: `channels moderated by the TC39` remove the "the" as all the other listed items don't use "the TC39"

The "In addition, violations of this code outside these spaces by a person others known to be a TC39 member is an act of representing TC39 may affect a person's ability to participate within them." sentence still doesn't sit very well IMO

I'm pretty sure this clause is here to try to stop someone from being racist on twitter, then wondering why we ask them to not be a part of the project.

We need a further discussion about this before we try to rewrite it IMO

File: Enforcement.md [hunk 2]
```diff
@@ -1,58 +1,58 @@
...
+### The Code of Conduct Enforcement Subcommittee
 
-All responses to reports of conduct violations will be managed by a [Code of Conduct Committee][] ("the committee").
+All responses to reports of conduct violations will be managed by a [Code of Conduct Enforcement Subcommittee][] ("the subcommittee").
 
-TC39 will establish this committee, comprised of at least three members. One member will be designated chair of the committee and will be responsible for all reports back to the board. The board will review membership on a regular basis.
+TC39 will establish this subcommittee, comprised of at least three members. One member will be designated chair of the subcommittee and will be responsible for all reports back to TC39. TC39 will review membership on a regular basis.
 
-### How the Committee Will Respond to Reports
+### How the subCommittee Will Respond to Reports
```
Comment (gnarf) [inline Enforcement.md]: weird caps

File: Enforcement.md [hunk 3]
```diff
@@ -1,58 +1,58 @@
...
 
-Finally the committee will make a report of the TC39 members and respective delegates and the ECMA secretariat. The committee will never publicly discuss the issue; all public statements will be made by the TC39 Chair and/or the ECMA Secretariat.
+Finally the subcommittee will make a report of the TC39 members and respective delegates and the ECMA secretariat. The subcommittee will never publicly discuss the issue; all public statements will be made by the TC39 Chair and/or the ECMA Secretariat.
 
 ### Conflicts of Interest
 
-In the event of any conflict of interest a committee member must immediately notify the other members, and recuse themselves if necessary.
+In the event of any conflict of interest - i.e., members who are personally connected to a situation - a subcommittee member must immediately notify the other members, and recuse themselves if necessary.
```
Comment (gnarf) [inline Enforcement.md]: I think we can get rid of "if necessary"

File: Conduct.md (hunk 1 above)
Comment (ljharb) [inline Conduct.md]: I agree - there does not exist a space where toxic behavior can exist in isolation. If you're a jerk anywhere, it should have consequences everywhere.

File: Conduct.md [hunk 4]
```diff
@@ -1,16 +1,16 @@
...
 
 If you believe someone is violating the Code of Conduct, we ask that you report it by emailing [???][]. For more details, please see our [Reporting Guidelines][].
 
 ### Be friendly and patient
 
-We understand that everyone has different levels of experience or knowledge in many diverse fields, be it technical or non-technical in nature. We also have areas of knowledge we are eager to expand; we want to be a community where people can not only contribute, but feel comfortable to ask questions as well and learn along the way. If someone says something wrong, or says something accidentally offensive, respond with patience and try to keep it polite and civil. Remember that we all were newbies at one point.
+We understand that everyone has different levels of experience or knowledge in many diverse fields, be it technical or non-technical in nature. We also have areas of knowledge we are eager to expand; we want to be a community where people can not only contribute, but feel comfortable to ask questions as well and learn along the way. If you're offended and decide to let the offender know, please do so in a polite and civil manner. Remember that we all were newbies at one point.
```
Comment (ajklein) [inline Conduct.md]: I get what @gnarf was going for, but this new sentence still seems oddly worded/placed ("If you're offended and decide to let the offender know..."). I like the second half of the original sentence ("respond with patience and try to keep it polite and civil"). It's the first I have a problem with. How about:

> When correcting another participant, respond with patience and try to keep it polite and civil.

File: Conduct.md (hunk 1 above)
Comment (leobalter) [inline Conduct.md]: I am rewording this to: 
 
> This Code of Conduct is enforced within all spaces managed by TC39. This includes IRC channels moderated by TC39, mailing lists as the es-discuss, issue trackers on projects hosted by TC39, TC39 events as the meetings and others hosted by or representing TC39.

The main change is `s/applies to/enforced within`.

**Comment by leobalter** [comment]
fixes are up.

**Comment by leobalter** [comment]
merging this to avoid more ambiguity. I willing fix anything necessary as a follow up

**Pull request merged**